prp/parse/qc.py

```diff
@@ -56,5 +215,21 @@ def parse_postalignqc_results(input_file: File) -> QcMethodIndex:
         mapped_reads=int(qc_dict["mapped_reads"]),
         tot_reads=int(qc_dict["tot_reads"]),
         iqr_median=float(qc_dict["iqr_median"]),
+        quartile1=float(qc_dict["quartile1"]),
+        median=float(qc_dict["median"]),
+        quartile3=float(qc_dict["quartile3"]),
     )
     return QcMethodIndex(software=QcSoftware.POSTALIGNQC, result=qc_res)
+
+
+def parse_alignment_results(sample_id: str, bam: File, reference: File, cpus: int, output: File, bed: File = None, baits: File = None) -> None:
+    """Parse bam file and extract relevant metrics.
+
+    Returns:
+        None
+    """
```

Comment:
```suggestion
    """
```

You can remove this, as it's already spelled out above.